cache/__packages

```diff
@@ -0,0 +1,20 @@
+base
```

Comment:
Hmm, did a few files leak into the top level of the repo? looks like cache/, figure/, and xaringan-themer.css made a break for it on their own! 😝 

A commit to remove those files would be helpful!

Reply:
oooops! my bad. i think my update should be good, but plz review